src/libraries/Common/src/System/Security/Cryptography/SlhDsa.cs

```diff
@@ -677,25 +816,55 @@ public int ExportSlhDsaSecretKey(Span<byte> destination)
             return secretKeySizeInBytes;
         }
 
+        /// <summary>
+        ///   Exports the current key in the FIPS 205 secret key format.
+        /// </summary>
+        /// <returns>
+        ///   The secret key.
```

Comment:
Similar to public key, this needs more words to avoid basically just saying `<returns>The answer.</returns>`